content/blog/a-summary-of-the-july-retrospectives/index.md

```diff
@@ -0,0 +1,98 @@
+| aarch64 Mac | JDK 11 | 6 | Bottlenecked on having only 1 machine for TCK | 2nd machine now added to temurin-compliance project |
+| arm32 Linux | JDK 8 | 11 | Unclear, possibly slow to add an owner to it, also short-handed, team vacations | Close follow-up to ensure ownership early |
+| x86 Solaris | JDK 8 | 10 | Delayed on packaging issue | Issue now fixed, [temurin-build/3061](https://github.com/adoptium/temurin-build/issues/3061) |
+| sparcv9 Solaris | JDK 8 | 20 | Considered a best effort platform, team would/should prioritize this last | No specific action required, Reduction in effort on other platforms, and no respins would mean these lower priority platforms get released quicker |
+
+What is encouraging about this list is that most of these issues have actionable responses, many of them completed during the release, as evident in our improved times for the respins.
+
+## New Features Need New Tests
+A project that is changing, is a project that is going to occasionally break.  This is will always be the case, but how do we pro-actively protect ourselves in the tug-of-war between adding new features and remaining stable and secure?  The answer is of course 'testing'.  While we have an ever-evolving suite of tests for verifying the OpenJDK binaries we produce via the [AQAvit project](https://adoptium.net/aqavit), we additionally need to be better at adding tests to verify the changes we are making to our own build and distribution scripts.
+
+Several changes went in ahead of the July CPU that introduced delays during the release.  2 of the changes are enhancements as part of our Secure Software Development Framework [(SSDF) activities](https://github.com/adoptium/adoptium/issues/120), [GPG signing](https://blog.adoptium.net/2022/07/gpg-signed-releases/) and the [addition of SBOM artifacts](https://github.com/adoptium/temurin-build/issues/2900). Both of these features introduce new artifacts that can be downloaded alongside of the JDK binaries.  
+
+During the release, there were some initial missteps when not all of the new artifacts were published.  While this was easily rectified, it prompted the team to add checks for ensuring we are publishing the expected set of artifacts.   
+
+A [Windows compiler upgrade](https://github.com/adoptium/temurin-build/pull/2992) was also attempted prior to the release, there were unwanted side-effects, like missing libraries.  These went unnoticed because we had no tests for verifying the full set to be produced, and other testing did not spot the missing libraries either, because our test machines have the compiler installed in order to compile native test material.  The immediate action was to [revert the change](https://github.com/adoptium/temurin-build/pull/3009), until the core issue is addressed.  To future-proof against similar problems, we will be adding smoke tests to be run in a stripped down environment.
```

Comment:
I've chaned `like` to `in this case a` in this suggestion which you can take or leave but I think it's worth linking to the issue so people can see the specifics of the sort of thing we can hit:

```suggestion
A [Windows compiler upgrade](https://github.com/adoptium/temurin-build/pull/2992) was also attempted prior to the release, there were unwanted side-effects - in this case a [missing runtime library](https://github.com/adoptium/temurin-build/issues/3052).  These went unnoticed because we had no tests for verifying the full set to be produced, and other testing did not spot the missing libraries either, because our test machines have the compiler installed in order to compile native test material.  The immediate action was to [revert the change](https://github.com/adoptium/temurin-build/pull/3009), until the core issue is addressed.  To future-proof against similar problems, we will be adding smoke tests to be run in a stripped down environment.
```